tests/main/interfaces-log-observe/task.yaml

```diff
@@ -44,3 +89,30 @@ execute: |
         echo "System log shouldn't be accessible"
         exit 1
     fi
+
+    echo "Or make ANY privileged syslog(2) calls"
+    log-observe-consumer.klogctl SYSLOG_ACTION_CLOSE | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_OPEN | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_READ | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_READ_CLEAR | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_CLEAR | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_CONSOLE_OFF | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_CONSOLE_ON | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_CONSOLE_LEVEL | MATCH 'ERROR (operation not permitted|permission denied)'
+    log-observe-consumer.klogctl SYSLOG_ACTION_SIZE_UNREAD | MATCH 'ERROR (operation not permitted|permission denied)'
+    # It is important that the following matches EPERM/EACCES, not EINVAL, as it
+    # is a stand-in for potential future SYSLOG_ACTIONs
+    log-observe-consumer.klogctl SYSLOG_ACTION_BAD | MATCH 'ERROR (operation not permitted|permission denied)'
```

Comment:
EACCESS here

Reply:
SYSLOG_ACTION_READ and SYSLOG_ACTION_SIZE_UNREAD are denied with EPERM because they are allowed by seccomp but denied by lack of CAP_SYSLOG (similar to the SYSLOG_ACTION_READ_ALL and SYSLOG_ACTION_SIZE_BUFFER on dmesg_restrict systems), but will make sure each action is targeting the appropriate error